Dockerfile

```diff
@@ -1,8 +1,10 @@
-FROM codingame/java-compiler
+FROM codingame/maven3-builder
 
 # Copy files
-COPY target/junit-runner-0.0.1-SNAPSHOT-jar-with-dependencies.jar /usr/src/codingame/junit-runner/junit-runner.jar
+COPY target/java-maven3-junit4-runner-0.0.1-SNAPSHOT-jar-with-dependencies.jar /usr/src/codingame/junit-runner/junit-runner.jar
 COPY src/main/resources/junit-runner /usr/src/codingame/junit-runner/
```

Comment:
The `junit-runner` script does not need to be in the generated jar file. So it could be moved from the `src/main/resources` directory.